lil_aretomo/utils.py

```diff
@@ -14,13 +14,12 @@ def prepare_output_directory(
         tilt_angles: List[float],
         basename: str,
         pixel_size: float,
-        skip_if_completed: bool
 ) -> Tuple[Path, Path]:
     """Create an output directory and write input files for AreTomo."""
     directory.mkdir(exist_ok=True, parents=True)
 
     tilt_series_file = directory / f'{basename}.mrc'
-    if not tilt_series_file.exists() or skip_if_completed is False:
+    if not tilt_series_file.exists():
```

Comment:
this line in general feels like it has potential to cause bugs, let's remove it and always write the TS